README.md

```diff
@@ -169,6 +211,34 @@ _Example_
 ];
 ```
 
+#### `asset definition`
+
+An object containing both metadata and an asset feed to be published on the asset server.
+
+_Example_
+
+```js
+{
+    tag: 'my-tag', // unique tag to identify all assets sent from this source.
+    type: 'js', // js or css
```

Comment:
not separate endpoints anymore?

Reply:
no, single endpoint now.

Reply:
Why? Inconsistent API...

Reply:
I have no preference one or the other way, but we should be consistent

Reply:
I guess my feeling was that with the advent of O.B. we would likely deprecate the older endpoints. I'd have replaced them but I'd prefer to introduce this feature in a backwards compatible way to minimize any pain associated with introducing it. If you buy the idea that we'll eventually deprecate the older endpoints then it makes sense to rethink and improve the API. My feeling is that this is a simplification and improvement since we no longer need to have whole pathways dedicated to the small difference of js vs css. That was my rationale at least. 

I'm sellin', you buyin'? ...Or not so much?

Reply:
If the plan is to drop `/js|css` then I'm down with this.